doc/parameters.md

```diff
@@ -524,6 +524,13 @@ directive](https://nginx.org/en/docs/http/ngx_http_core_module.html#large_client
 
 Sets the maximum size of shared memory used by batcher policy. The accepted [size units](https://github.com/openresty/lua-nginx-module?tab=readme-ov-file#lua_shared_dict) are k and m.
 
+### `APICAST_PROXY_BUFFER_SIZE`
+
+**Default:** 4k|8k;
+**Value:** string
+
+Sets the size of the buffer used for handling the response received from the proxied server. This variable will set both [`proxy_buffer` NGINX directive](https://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_buffers) and [`proxy_buffer_size` NGINX directive](https://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_buffer_size). By default, the buffer size is equal to one memory page. This is either 4K or 8K, depending on a platform.
```

Comment:
```suggestion
Sets the size of the buffer used for handling the response received from the proxied server. This variable sets both [`proxy_buffer` NGINX directive](https://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_buffers) and [`proxy_buffer_size` NGINX directive](https://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_buffer_size). By default, the buffer size is equal to one memory page. This is either 4K or 8K, depending on a platform.
```

Reply:
What unit does the K stand for?

Reply:
sorry for my ignorance, but aren't they the same? ie 1024bytes? Actually the above sentence was copied from Nginx docs 
https://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_buffer_size

Reply:
The reason I asked for clarification is that the IBM Style Guide "distinguishes between uppercase K
(meaning 1024, or 2 to the power of 10) and lowercase k (meaning 1000, or 10 to the power of 3). Therefore, KB means 1024 bytes, and kB means 1000 bytes" for kilobyte, and therefore we must use either KB (1024 bytes) or kB (1000 bytes) depending on the meaning of 4 K or 8 K in the documentation.

However, we would write kibibytes as KiB.

Sorry for the confusion.

Reply:
@dfennessy updated to use KiB as you suggested. If you are happy with this change, could you please click that :white_check_mark:  button? :smile:

Reply:
Hi @tkan145 It wasn't my intention to muddy the waters. Thank you for your understanding :smile: 

I'll just to reiterate, KB means 1024 bytes, and kB means 1000 in kilobytes.

If you're sure you want to use kibibytes (KiB), where KiB equals 1024 bytes, then I approve.

Either if the above I approve as long as it works for the reader in their understanding :nerd_face: